packages/vite/src/node/build.ts

```diff
@@ -133,6 +135,22 @@ export interface BuildOptions {
    * @default minify
    */
   cssMinify?: boolean
+  /**
+   * @experimental
+   * Use Lightning CSS to minify CSS. This should be installed as a peer
+   * dependency.
+   * @default esbuild
+   */
+  cssMinifier?: 'esbuild' | 'lightningcss'
+  /**
+   * @experimental
+   * Targets is automatically converted from esbuild (css)target, but
+   * can be provided for more granular control.
+   */
+  lightningcss?: {
+    targets?: LightningCSS['Targets']
+    drafts?: LightningCSS['Drafts']
+  }
```

Comment:
This idea is to be able to use lightningcss as a minifier without and keep postcss for transformation. 
And I also the conversion from `build.cssTarget` is not 1-1 so that's nice to keep it possible IMO

Reply:
Maybe my comment above (https://github.com/vitejs/vite/pull/12807#discussion_r1225843113) with `css.lightningcss.target` would help with this.